Sources/Bot/Models/IssueComment.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+public struct IssueComment: Equatable {
+    let user: GitHubUser
+    let body: String
+    let creationDate: Date
+}
+
+extension IssueComment: Decodable {
+    private static let dateFormatter: DateFormatter = {
+        // 2020-03-30T11:26:18
+        let df = DateFormatter()
+        df.locale = Locale(identifier: "en_US_POSIX")
+        df.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZ"
+        return df
+    }()
+
+    private enum CodingKeys: String, CodingKey {
+        case user
+        case body
+        case created_at
+    }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        self.user = try container.decode(GitHubUser.self, forKey: .user)
+        self.body = try container.decode(String.self, forKey: .body)
+        let dateString = try container.decode(String.self, forKey: .created_at)
+        guard let date = Self.dateFormatter.date(from: dateString) else {
```

Comment:
no way to use date decoding strategy?

Reply:
Oh forgot about that one, good point.
And just saw that there's actually already code for that in `Decoding.swift` so all that shouldn't even be needed at all indeed.

Reply:
Should be fixed in 33d971a3d08332d8f1e7465720593ffc6e78a067 👍